src/node_redis/index.ts

```diff
@@ -3,9 +3,7 @@ import { Commands } from "../generated/interface";
 import { flattenDeep } from "../flatten";
 import { WrappedNodeRedisMulti, WrappedNodeRedisMultiImpl } from "./multi";
 
-declare module "redis" {
-    export const addCommand: (name: string) => void;
-}
+declare module "redis" {}
 
```

Comment:
Seems like there's no need for this anymore

```suggestion
```

Reply:
Donezo